src/cpp/session/CMakeLists.txt

```diff
@@ -625,13 +625,6 @@ if(APPLE)
          add_custom_command (TARGET rsession POST_BUILD
             COMMAND install_name_tool -change ${LIB_DIR}/${LIB_FILE} @executable_path/../Frameworks/${LIB_FILE} ${CMAKE_CURRENT_BINARY_DIR}/rsession)
       endforeach()
-
-      # also rewrite where rsession looks for libR.dylib
-      # (so we don't accidentally load the wrong version of R at launch)
-      get_filename_component(LIBR_HOME_REALPATH "${LIBR_HOME}" REALPATH)
-      add_custom_command(TARGET rsession POST_BUILD
-         COMMAND install_name_tool -change "${LIBR_HOME_REALPATH}/lib/libR.dylib" "@executable_path/libR.dylib" "${CMAKE_CURRENT_BINARY_DIR}/rsession")
```

Comment:
We could consider whether rewriting this to be `/Library/Frameworks/R.framework/Versions/Current/Resources/lib/libR.dylib`?

Reply:
(assuming we're allowed to have library link paths outside of the application bundle)

Reply:
Good idea, will give that a shot.

Reply:
Update: works pretty well!

Reply:
🎉 🎉 🎉 🎉 🎉